gamify-specs/src/test/java/ch/heigvd/gamify/api/spec/defs/BadgesDefs.java

```diff
@@ -0,0 +1,85 @@
+package ch.heigvd.gamify.api.spec.defs;
+
+import ch.heigvd.gamify.ApiException;
+import ch.heigvd.gamify.api.BadgesApi;
+import ch.heigvd.gamify.api.dto.Badge;
+import ch.heigvd.gamify.api.spec.env.Environment;
+import io.cucumber.java8.En;
+
+public class BadgesDefs implements En {
```

Comment:
Please format this whole file automatically (and use Google Style for all the code in general, it's not interesting to talk about formatting issues in PRs)